### ISSUE TITLE
Set Vaxrank params

### ISSUE THREAD

**Issue by ihodes** [hammerlab/epidisco#41]

**Comment by iskandr** [comment]
Differences from the defaults in Biokepi:

``` ocaml
padding_around_mutation = 5; (* previous 0 *)
max_vaccine_peptides_per_mutation = 3; (* previously 1 *)
max_mutations_in_report = 20; (* previously 10...though we want to only use the top 10, we should probably report beyond that *)
```

**Comment by ihodes** [comment]
fixed in #46
